testsuite/features/step_definitions/smdba_steps.rb

```diff
@@ -1,6 +1,9 @@
-# Copyright 2011-2022 SUSE LLC.
+# Copyright 2011-2023 SUSE LLC.
 # Licensed under the terms of the MIT license.
 
+### This file contains all step-definitions concerning configuration of
```

Comment:
all step definitions
or
the definition of all steps
(like in the other files)